Description: moving boost thread include out of header that does not use it
 Patch is not yet forwarded upstream.
Author: Laszlo Kajan <lkajan@rostlab.org>
--- a/src/utils.h
+++ b/src/utils.h
@@ -10,7 +10,6 @@
 #endif
 
 #include <time.h>
-#include <boost/thread.hpp>
 #include <boost/lexical_cast.hpp>
 #include <boost/format.hpp>
 #include <boost/filesystem.hpp>
--- a/src/structure.cpp
+++ b/src/structure.cpp
@@ -18,6 +18,7 @@
 #define foreach BOOST_FOREACH
 #include <boost/algorithm/string.hpp>
 #include <boost/math/special_functions/round.hpp>
+#include <boost/thread.hpp>
 
 #include "align-2d.h"
 #include "utils.h"
